www/main/banidb/realm-search.js

```diff
@@ -49,7 +49,7 @@ const init = () => {
  * search('jggsspp', 0, 'all');
  * // => [{ Gurmukhi: 'jo gurisK guru syvdy sy puMn prwxI ]', ID: 31057 },...]
  */
-const query = (searchQuery, searchType, searchSource) =>
+const query = (searchQuery, searchType, searchSource, howManyRowsToShow = 20) =>
```

Comment:
Not really a bug, but was thinking if we can we rename this state name to something like `resultRows` ? It would be more clearer imo.

Reply:
Done veerji